zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/variable/ClusterVariableRecord.java

```diff
@@ -28,8 +28,7 @@ public class ClusterVariableRecord extends UnifiedRecordValue
   private final LongProperty variableKeyProp = new LongProperty(VARIABLE_KEY_KEY, 0L);
```

Comment:
❓ Any reason why the key is needed? Can this be deleted including `#getKey()`? If it is needed, ensure to set it accordingly the processor.